lp-base/tasks/main.yml

```diff
@@ -14,6 +14,9 @@
 - include: packages.yml
   tags: [ 'packages' ]
 
+- include: cpu.yml
```

Comment:
Consider introducing a `when` clause on an include instead, DRY and more readable.